frontend/src/EventDetails/EventDetails.module.css

```diff
@@ -0,0 +1,102 @@
+.container {
+  background: hsl(0, 0%, 100%);
+  border-radius: 16px;
+  padding: 0;
+  padding-bottom: 40px;
+  width: 76vw;
+  min-width: 700px;
+  min-height: fit-content;
+  height: 90vh;
+  box-shadow: 0px 0px 16px hsla(0, 0%, 0%, 0.25);
+}
+
+.picture {
+  width: 100%;
+  border-radius: 16px 16px 0px 0px;
+  height: 280px;
+  background-size: cover;
+}
+
+.info {
+  display: flex;
+  padding-left: 40px;
+  padding-right: 40px;
+  min-height: fit-content;
+  height: 360px;
+}
+
+.name {
+  font-size: 48px;
```

Comment:
⚠️ action required: can you represent this using rem units?
![image](https://github.com/user-attachments/assets/ac1617a2-8d1f-4f34-b51e-6dd334278fe5)